RecoTracker/FinalTrackSelectors/python/MergeTrackCollections_cff.py

```diff
@@ -33,7 +33,8 @@
 # This customization will be removed once we get the templates for
 # phase2 pixel
 from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
-phase2_tracker.toModify(duplicateTrackCandidates, ttrhBuilderName = "WithTrackAngle") # FIXME
+phase2_tracker.toModify(duplicateTrackCandidates, ttrhBuilderName = "WithTrackAngle", forestLabel = None, minBDTG = -100.)
```

Comment:
Another possible source of differences wrt baseline?

Reply:
Indeed, it is this line. I will revert this change, thanks for the suggestion.